kubernetes/loculus/templates/_config-processor.tpl

```diff
@@ -52,6 +52,16 @@
         secretKeyRef:
           name: orcid
           key: orcidSecret
+    - name: LOCULUSSUB_s3accessKey
+      valueFrom:
+        secretKeyRef:
+          name: s3-bucket
+          key: accessKey
+    - name: LOCULUSSUB_s3secretKey
+      valueFrom:
+        secretKeyRef:
+          name: s3-bucket
+          key: secretKey
```

Comment:
```suggestion
    - name: LOCULUSSUB_s3accessKey
      valueFrom:
        secretKeyRef:
          name: s3-bucket
          key: accessKey
    - name: LOCULUSSUB_s3secretKey
      valueFrom:
        secretKeyRef:
          name: s3-bucket
          key: secretKey
```

IIRC we discussed that we don't need/use these

Reply:
yes! thanks. https://github.com/loculus-project/loculus/pull/3978/commits/7d26f497c88572095a793fdca60362ce190a8462